### PR TITLE
notebook stopper

### PR DESCRIPTION
**Pull request by slemasne**
# Pull Request Checklist

- [ ] Read the [contributing guidelines](https://github.com/finbourne/lusid-python-tools/blob/master/docs/CONTRIBUTING.md)
- [ ] Tests pass

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

File: lusidtools/jupyter_tools/notebook_stopper.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class StopExecution(Exception):
+    """
+    This class is used to stop a notebook from running.
+    A sample implementation - stop the notebook if a certain portfolio does not exist
```
Comment (rizwansaeed) [inline lusidtools/jupyter_tools/notebook_stopper.py:4]: I would remove this line and give an example of how to use it e.g. `raise StopExecution("missing portfolio")`

File: lusidtools/jupyter_tools/notebook_stopper.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class StopExecution(Exception):
```
Comment (rizwansaeed) [inline lusidtools/jupyter_tools/notebook_stopper.py:1]: maybe rename the file to `stop_execution.py` so its easier to find/know what is in the file

File: lusidtools/jupyter_tools/notebook_stopper.py (hunk 1 above)
Comment (slemasne) [reply lusidtools/jupyter_tools/notebook_stopper.py]: Done.
See

Comment (slemasne) [reply lusidtools/jupyter_tools/notebook_stopper.py]: https://github.com/finbourne/lusid-python-tools/pull/131

File: lusidtools/jupyter_tools/notebook_stopper.py (hunk 2 above)
Comment (slemasne) [reply lusidtools/jupyter_tools/notebook_stopper.py]: Done
See https://github.com/finbourne/lusid-python-tools/pull/131

**Pull request closed without merging**